### PR TITLE
Add security section to Call REST service

### PR DESCRIPTION
**Pull request by sbont**

### CODE REVIEW

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md [hunk 1]
```diff
@@ -249,3 +249,22 @@ There are two ways to resolve this:
 2. Handle the error in your microflow and retry a number of times before returning the error. Your flow might look similar to the one below.
 
     {{< figure src="/attachments/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action/retry-rest-connection-timeout.png" >}}
+
+## 10 Security Considerations {#security}
+
+### 10.1 Preventing Vulnerabilities in your App
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: Do we need a subheading here? It seems a bit strange while this is the only security consideration we are talking about.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md [hunk 2]
```diff
@@ -249,3 +249,23 @@ There are two ways to resolve this:
...
+
+To prevent this type of attack from happening, you can take the following precautions:
+
+* Verify that your Call REST service activities never call a URL that is input by the user.
+
+If that is not possible:
+
+* Always validate and sanitize the user inputs
+* Maintain a whitelist of the domains that should be accessible and use it to validate URLs
+* Do not allow the user to access an unprocessed response from the call
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: Do we need to mention that a (malicious) user can interrogate the objects which are passed between the runtime client and the runtime server, so it is not necessarily enough not to let the user see the information on a page, but it mustn't be retained (e.g. in a Non-Persistable entity) at the end of a transaction with the user?
https://docs.mendix.com/refguide/communication-patterns/

**Review by MarkvanMents — COMMENTED** [review]
Thanks for the PR, Steven (@sbont). Looks like great content.

I've made some proofreading changes and asked a couple of questions. But, if you are happy with my proofreading then I can approve and merge.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md (hunk 1 above)
Comment (sbont) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: Fair, I'll come up with a better heading

Comment (sbont) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: What do you think of the last addition? I kept the heading and subheading, as "security considerations" is too general for the content whereas "preventing SSRF" is too technical / detailed for a heading, I feel. So I've added a bit of introduction to help the reader understand what it is about (and if it's relevant to them).
Important background here is that we need to document this specific SSRF attack vector as part of how we deal with security vulnerabilities in our platform and complying with certifications (I believe FedRAMP specifically)

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md (hunk 2 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: Hi Steven, @sbont 
Is this something you need to address, or shall I publish this PR now?

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: Thanks Steven.
It's generally a bit strange to have just one subsection, but I think it makes sense here. So lets keep it like it is.

File: content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md (hunk 2 above)
Comment (sbont) [reply content/en/docs/refguide/modeling/application-logic/microflows-and-nanoflows/activities/integration-activities/call-rest-action.md]: Good point, will add

**Review by MarkvanMents — APPROVED** [review]
Approved in discussion over Slack.
Approving and merging.

**Pull request merged**